package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@dsb-norge/vue-keycloak-js",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

We update this when we release, please revert.

```suggestion
  "version": "1.1.2",
```